.github/workflows/build-validation.yml

```
@@ -19,6 +19,21 @@ jobs:
       - name: Run Prebuild validations
         run: echo "Secret detections and other prebuild validation would be executed here (customize as needed)."
 
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)